planetiler-core/src/main/java/com/onthegomap/planetiler/util/YAML.java

```diff
@@ -33,12 +36,48 @@ public static <T> T load(Path file, Class<T> clazz) {
   public static <T> T load(InputStream stream, Class<T> clazz) {
     try (stream) {
       Object parsed = snakeYaml.loadFromInputStream(stream);
+      handleMergeOperator(parsed);
       return convertValue(parsed, clazz);
     } catch (IOException e) {
       throw new UncheckedIOException(e);
     }
   }
 
+  /**
+   * SnakeYaml doesn't handle the <a href="https://yaml.org/type/merge.html">merge operator</a> so manually post-process
+   * the parsed yaml object to merge referenced objects into the parent one.
+   */
+  private static void handleMergeOperator(Object parsed) {
+    if (parsed instanceof Map<?, ?> map) {
+      Object toMerge = map.remove("<<");
```

Comment:
Could this code, or similar, be used check if the merge operator is the first key in the mapping?!

```suggestion
    if (parsed instanceof Map<String, Object> map) {
      // Check if the first key is a merge operator using an iterator
      Iterator<String> keyIterator = parsed.keySet().iterator();
      Object toMerge = null;
      if (keyIterator.hasNext() && 
          keyIterator.next().equals("<<")) {
            toMerge = parsed.remove("<<");
      }
```